src/kaleidoscope/plugin/Turbo.cpp

```diff
@@ -0,0 +1,99 @@
+/* -*- mode: c++ -*-
+ * Kaleidoscope-Turbo
+ * Copyright (C) 2018 ash lea
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <Kaleidoscope-Turbo.h>
+#include <Kaleidoscope-LEDControl.h>
+
+namespace kaleidoscope {
+namespace plugin {
+uint16_t Turbo::interval = 50;
+uint16_t Turbo::flash_interval = 69;
+bool Turbo::toggle = false;
+bool Turbo::flash = true;
+cRGB Turbo::activeColor = (cRGB) {
+  0x00, 0x00, 0xff
```

Comment:
```suggestion
  0, 0, 160
```

For power consumption reasons, we usually use dimmer colors than full-bright. This'd adjust the highlight color to the same as `solidRed` in the factory firmware.